root/components/InstrumentRelTypes.js

```diff
@@ -30,7 +30,10 @@ const InstrumentRelTypes = ({
             let listElement = l_relationships(relType.name);
             if (relType.credit) {
               listElement = listElement + ' ' +
-                bracketedText(texp.l('as “{credit}”', {credit: relType.credit}));
+                bracketedText(texp.l(
+                  'as “{credit}”',
+                  {credit: relType.credit},
+                ));
```

Comment:
Related to this silent eslint change, `/artists` gives me:
```
Couldn't render template "undef error - Error: No translation key found.
    at Jed.dcnpgettext (/musicbrainz-server/node_modules/jed/jed.js:169:13)
    at Jed.dgettext (/musicbrainz-server/node_modules/jed/jed.js:85:16)
    at eval (webpack:///./root/static/scripts/common/i18n/wrapGettext.js?:56:20)
    at eval (webpack:///./root/components/InstrumentRelTypes.js?:23:23)
    at Array.map (<anonymous>)
    at InstrumentRelTypes (webpack:///./root/components/InstrumentRelTypes.js?:21:290)
    ...
```
even after rebuilding `po/mb_server.pot` and recompiling resources.

Reply:
Not related to eslint, just to me being an idiot (the line breaking was the one above, because I was trying to check relType.name but the actual object had relType.role - fixed).

Reply:
Thank you, it works for `/artists` now but all other tabs started to give that error.

Reply:
Sigh, I'm dumb. Ok, should work now.

Reply:
Thank you, still silent coding style change isn’t a good thing, please avoid these.